fga-eps-mds/2020.1-stay-safe-front-end
# Add push notifications to app

## Issue 
Resolves #74 


## Description

This PR setups a notification context that allows the API to send push notifications to users.

## Checklist:



## How to validate?

You will need to use the *56-notifications* branch from 2020.1-stay-safe-user-service repository and login on Expo with ```expo login``` on terminal. Logout and login on application, wait for 10 seconds and check if you receive a welcome custom notification of Stay Safe group .
